internal/layers/layers.go

```diff
@@ -65,6 +65,10 @@ func (lc *lcowLayersCloser) Release(ctx context.Context) (retErr error) {
 // UVM at which container scratch directory is located. Usually, this path is the path at which the container
 // scratch VHD is mounted. However, in case of scratch sharing this is a directory under the UVM scratch.
 func MountLCOWLayers(ctx context.Context, containerID string, layerFolders []string, guestRoot string, vm *uvm.UtilityVM) (_, _ string, _ resources.ResourceCloser, err error) {
+	if vm == nil {
+		return "", "", nil, errors.New("MountLCOWLayers cannot be called for process-isolated containers")
+	}
+
```

Comment:
Does this happen in the tests? :o

Reply:
no, but i figured it'd make it match `MountWCOWLayers`, and better safe than sorry 😅